source/js/buyers-guide/components/social-share/social-share.scss

```diff
@@ -0,0 +1,47 @@
+.pni-share-buttons {
```

Comment:
It'd better if we let elements adjust width themselves instead of setting fixed width and margin. I would simply the following SCSS to

```scss
.pni-share-buttons {
  max-width: 568px; // width from comps
  margin: 0 auto;

  .pni-s-link {
    $margin-x: 0.5rem;

    border: 1px solid black;
    border-bottom-width: $btn-shadow-width;
    color: black;
    transition: all 0.1s linear;
    border-radius: 2em;
    white-space: nowrap;
    margin-bottom: 1rem;
    padding: 0.5em 0;
    font-weight: 600; // this should be semi-bold so 600
    flex: 1;

    @media screen and (max-width: $bp-md) { // Generally we want to avoid using "max-width" since we write our SCSS in mobile first style. This is an exception here since we want these rulesets to apply only on mobile and using "max-width" is the cleanest solution here.
      &:nth-child(odd) {
        margin-right: $margin-x;
      }

      &:nth-child(even) {
        margin-left: $margin-x;
      }
    }

    @media screen and (min-width: $bp-md) {
      margin-left: 0.5rem;
      margin-right: 0.5rem;

      &:first-child {
        margin-left: 0;
      }

      &:last-child {
        margin-right: 0;
      }
    }

    @include hover-focus-active {
      ...
```
(don't forget to remove my inline comments)

Reply:
# 🔷 mobile (buttons are now wider and they fill the row)
<img src="https://user-images.githubusercontent.com/2896608/52381588-e4f69580-2a26-11e9-9c9c-d0e800e85276.png" width="320" />

# 🔷 desktop
![screen shot 2019-02-06 at 15 41 50](https://user-images.githubusercontent.com/2896608/52381590-e6c05900-2a26-11e9-9638-17d75fdcd315.png)

Reply:
We'll still have to adjust the icon sizes and position since they differ quite a bit from comps. I can take this on and tackle it as a followup since it requires access to Sketch 👀

Reply:
Could I ask you to just commit that on this branch?

Reply:
Oops, 2 mins. I think my suggested code above can be simplified further.

Reply:
let's do this in follow-up and get this in today still

Reply:
follow-up filed: https://github.com/mozilla/foundation.mozilla.org/issues/2618